pkg/client/e2e.go

```diff
@@ -44,9 +45,9 @@ func (*SonobuoyClient) GetTests(reader io.Reader, show string) ([]reporters.JUni
 			}
 			// TODO(chuckha) consider reusing this function for any generic e2e-esque plugin results.
 			// TODO(chuckha) consider using path.Join()
-			if path == e2eJunitPath {
+			if path == e2eJunitPath || path == legacye2eJunitPath {
 				found = true
-				return results.ExtractFileIntoStruct(e2eJunitPath, path, info, &junitResults)
+				return results.ExtractFileIntoStruct(path, path, info, &junitResults)
```

Comment:
All of these `Extract...` functions are kind of hard to read IMO; I think in some cases we're misusing them and we don't need to be first filtering for the filepath. The only reason to do so here is to (1) mark that we actually found the file and (2) here we want to extract _either_ file into the struct instead of a hardcoded one.